### PR TITLE
Hoist HTTPStatus into falcon top-level namespace

### LINKED ISSUE #592: Hoist HTTPStatus into falcon top-level namespace
I.e., add an import line to `falcon/__init__.py`

### PR DESCRIPTION
**Pull request by maxking**
Fixes #592

### CODE REVIEW

File: falcon/__init__.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 from falcon.status_codes import *  # NOQA
 from falcon.errors import *  # NOQA
 from falcon.http_error import HTTPError  # NOQA
+from falcon.http_status import HTTPStatus # NOQA
```
Comment (kgriffs) [inline falcon/__init__.py]: Just needs an extra space before the `#`

**Comment by kgriffs** [comment]
@maxking Thanks for working on this! Once this passes the pep8 gate, could you please rebase and squash all the patches down into a single commit?

**Comment by maxking** [comment]
Thanks for the review. I will do that ASAP!

**Comment by maxking** [comment]
@kgriffs Done!

**Comment by kgriffs** [comment]
Oops, forgot to mention the commit message should follow AngularJS style (e.g., prefix the first line with a "chore:", but not a big deal this time; just something to keep in mind for the future. Thanks!

**Pull request merged**